packages/antify-admin/src/server/api/mail_templates/[mailTemplateId]/send_test_mail.post.ts

```diff
@@ -0,0 +1,69 @@
+import prisma from '~~/server/datasources/db/client';
+import { useGuard } from '~~/composables/useGuard';
+import { HttpForbiddenError } from '~~/server/errors';
+import { useAuthorizationHeader } from '~~/server/utils/useAuthorizationHeader';
+import { useTenantHeader } from '~~/server/utils/useTenantHeader';
+import { PermissionId } from '~~/server/datasources/static/permissions';
+import {
+  Input,
+  Response,
+  validator,
+} from '~~/glue/api/mail_templates/[mailTemplateId]/send_test_mail.post';
+import { useMailer } from '~~/server/utils/useMailer';
+
+export default defineEventHandler<Response>(async (event) => {
+  const guard = useGuard(useAuthorizationHeader(event));
+
+  if (!guard.isUserLoggedIn) {
+    throw new HttpForbiddenError();
```

Comment:
I would do this `if` and the next in one, it would look a bit cleaner and you're not doing anything different anyway:
```
if (!guard.isUserLoggedIn || !guard.hasPermissionTo(PermissionId.CAN_EDIT_MAIL_TEMPLATES, tenantId)) {
    throw new HttpForbiddenError();
}
```

Reply:
You're right! Code is updated